experimental/common-expression-language/README.md

```diff
@@ -1,91 +1,14 @@
-|${}        |Used to denote a variable in template expansion. E.g. ${myVariable}                        |N/A                                |
-|()            |Enforces precedence order and groups sub expressions into larger expressions. E.g. (A+B)*C    |N/A                                |
-|.            |Property selector. E.g. myObject.Property1                                                    |N/A                                |
-|\            |Escape character for templates, expressions.                                               |N/A                                |
-
-## Variables
-
-Variables are always referenced by their name in the format `${myVariable}`. Variables can be referenced either using the property selector operator in the form of `myParent.myVariable`, using the item index selection operator like in `myParent.myList[0]`, or using the [getProperty](./prebuilt-functions.md#getProperty) function. 
-
-There are two special variables, `[]` and  `{}`. `[]` represents an empty list and `{}` represents a empty object.
-
-## Explicit values
-
-Explicit values can be enclosed in either single quotes 'myExplicitValue' or double quotes "myExplicitValue".
-
-## Additional resources
-
-- [NuGet AdaptiveExpressions](https://www.nuget.org/packages/AdaptiveExpressions) package for C#
-- [npm adaptive-expressions](https://www.npmjs.com/package/adaptive-expressions) package for Javascript
-- [API reference](./api-reference.md) for Adaptive Expressions
-- [Pre-built functions](./prebuilt-functions.md) supported by the Adaptive Expressions library
-- [Extend functions](./extend-functions.md)
-
-[1]:prebuilt-functions.md#add
-[2]:prebuilt-functions.md#sub
-[3]:prebuilt-functions.md#mul
-[4]:prebuilt-functions.md#div
-[5]:prebuilt-functions.md#exp
-[6]:prebuilt-functions.md#mod
-[7]:prebuilt-functions.md#equals
-[8]:prebuilt-functions.md#not
-[9]:prebuilt-functions.md#greater
-[10]:prebuilt-functions.md#less
-[11]:prebuilt-functions.md#greaterOrEquals
-[12]:prebuilt-functions.md#essOrEquals
-[13]:prebuilt-functions.md#and
-[14]:prebuilt-functions.md#or
-[15]:https://botbuilder.myget.org/feed/botbuilder-declarative/package/nuget/Microsoft.Bot.Builder.Expressions
-[20]:https://github.com/microsoft/BotBuilder-Samples/blob/master/experimental/language-generation/README.md
+# Adaptive Expressions (**Stable release**)
+
+This feature is no longer experimental. It is available in **Stable release**, starting botbuilder SDK version 4.9.0 and higher.
+
+| Language     | Availability          | Docs         |
+|--------------|-----------------------|--------------|
+| csharp       | Generally available   | [here][1]    |
+| javascript   | Generally available   | [here][1]    |
+| python       | Not available         | N/A          |
+| java         | Not available         | N/A          |
+ 
+[1]:https://aka.ms/adaptive-expressions
```

Comment:
>ttps://aka.ms/adaptive-expressions [](start = 5, length = 34)

this has not been updated yet?

Reply:
No Tom. Do not have final doc location yet from the team. For now, this continue to point to itself. Will update the aka.ms link once I know the final doc location.